packages/models-library/src/models_library/groups.py

```diff
@@ -35,7 +38,47 @@ class Group(BaseModel):
         create_enums_pre_validator(GroupType)
     )
 
-    model_config = ConfigDict(populate_by_name=True)
+    @staticmethod
+    def _update_json_schema_extra(schema: JsonDict) -> None:
```

Comment:
Q: please remind me what is the benefit in doing this?

Reply:
some pros:
- better typing resolution (remember the `type: ignore` we had to put? )
- loaded only when needed (e.g. to build openapi doc)
- more flexibility to modify dynamically the entire schema (not only the examples part)